### DISCUSSION TITLE
Docker script cannot access portainer?

### DISCUSSION THREAD

**Discussion by Sale6666** [community-scripts/ProxmoxVE#639] [General]
Hi all im new to proxmox and started playing around with scripts and have installed some things such as heimdall, pihole, changedetection, gotify and all works fine out of the box however now i installed docker and chose to install portainer with it as it gives the option when running the script and i cannot access it? When i go to my local ip 192.168.0.44:8000 or even 9443 i get a small note up left in screen "not found" thats all or it simply does not connect.
Networking is the same i set static ip and i attempted also dhcp install all the same... 
What could i be doing wrong with this since i did not change anything different than other containers?
Thanks!

**Comment by MickLesk** [comment]
https://IPADRESS:9443/#!/2/docker/containers ?

**Reply by Sale6666** [reply]
Hi im not sure about what that would be but its like this: my proxmox is 192.168.0.99:8006 than the helper script installed docker with ip 192.168.0.44 and in the container i see a running portainer at 0.0.0.0:8000 or 9433... So everything else works as intended so should me going to 192.168.0.44:8000 be able to access portainer? Sorry im still a bit a noob at proxmox and LXC

**Reply by MickLesk** [reply]
Portainer running on 9443

**Reply by Sale6666** [reply]
> Portainer running on 9443

Hi i got it working now but the 9443 did not work either.
The issue might have been i used ubuntu 24.10 now i downloaded the 22.04.05 and done the exact same thing and it works as intended on 9443.
One more thing i noticed is when installing ubuntu 24.10 and you go and set network to static to continue it says continue without network… seems bugged a bit!
All working with older version 👍
